### PR TITLE
add PopService

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: proto/integration.proto [hunk 1]
```diff
@@ -447,4 +447,12 @@ message ForkresolutionConfigRequest{
 
 service ForkresolutionService{
     rpc CompareTwoBranches(TwoBranchesRequest) returns (CompareReply) {}
+}
+
+message EmptyReply{
+}
+
+service PopService{
+    rpc CheckVTBInternally(BytesArrayRequest) returns (EmptyReply) {}
```
Comment (blackyblack) [inline proto/integration.proto:456]: Does it make sense to return empty reply?

File: proto/integration.proto [hunk 2]
```diff
@@ -447,4 +447,12 @@ message ForkresolutionConfigRequest{
 
 service ForkresolutionService{
     rpc CompareTwoBranches(TwoBranchesRequest) returns (CompareReply) {}
+}
+
+message EmptyReply{
+}
+
+service PopService{
```
Comment (blackyblack) [inline proto/integration.proto:455]: What is so special about this service? Why is it separate?

**Comment by Mr-Leshiy** [comment]
WIP

File: proto/integration.proto (hunk 2 above)
Comment (Warchant) [reply proto/integration.proto:455]: we don't need to use most of current api calls. this service is 1x1 api to btc's service.

File: proto/integration.proto (hunk 1 above)
Comment (Warchant) [reply proto/integration.proto:456]: it should return bool, isn't it?

**Pull request closed without merging**